src/pages/components/progress-indicator/usage.mdx

```diff
@@ -45,41 +48,192 @@ set of steps, guiding toward the completion of a specified process.
+| Not started | A step is not started when a user has not yet interacted with that step. Steps the user has not encountered yet, or future steps, are indicated by an outlined circle and a gray active line.                                                                                                                                                 |
+| Error       | A step may be in error when a user has entered invalid or incomplete information. There could also be a server-side error. Provide clear information about the error and guidance on how to resolve the issue.                                                                                                                                |
+| Disabled    | A step is disabled when all interactive functions have been removed. Unlike read-only states, disabled states are not focusable, are not read by screen readers, and do not need to pass visual contrast, making them inaccessible if they need to be interpreted.                                                                            |
+| Hover       | Use hover state when a user's mouse cursor is hovering over the progress indicator's step.                                                                                                                                                                                                                                                    |
+| Focus       | When a user tabs to or clicks on the progress indicator's step, the step becomes focused, indicating the user has successfully navigated to the component.                                                                                                                                                                                    |
+
+### Interactions
+
+#### Mouse
+
+If the progress indicator is interactive, users can trigger a state change by
+clicking on the step label.
+
+#### Keyboard
+
+The first step should be selected by default. Users can navigate between steps
+by pressing right or left arrow keys. For additional keyboard interactions, see
+the [Accessibility](/components/progress-indicator/accessibility/) tab.
+
+### Validation
+
+When possible, use validation to confirm that a step has been completed before
+the user continues. If any entry is invalid, the progress indicator should show
+an error state. Additionally, the invalid entry should be marked with an error
+state and include an inline error message that helps the user understand the
+problem and how to fix it. This kind of inline validation (or client-side
+validation) should happen as soon as the field loses focus.
+
+If the user cannot proceed due to a server-side issue, then an
+[Inline Notification](/components/notification/code#inline-notification) should
+appear.
+
+<Row>
+<Column colLg={8}>
+
+![Example of progress indicator used with validation](images/progress-indicator-usage-5.png)
+
+</Column>
+</Row>
+
```

Comment:
We have not been consistent with this but I'd also add ### Clickable areas for the Interactive progress indicator. [Example](https://carbondesignsystem.com/components/tree-view/usage/#interactions) in TreeView